app/public/static/js/components/interactable.js

```diff
@@ -0,0 +1,42 @@
+AFRAME.registerComponent("interactable", {
+  schema: {
```

Comment:
Do you ever use these schema parameters? `axis` or `value`? Are they leftover code from before?

Reply:
it was leftover lol